src/libostree/ostree-bootloader-syslinux.c

```diff
@@ -84,12 +92,13 @@ append_config_from_loader_entries (OstreeBootloaderSyslinux  *self,
       if (regenerate_default && i == 0)
         g_ptr_array_add (new_lines, g_strdup_printf ("DEFAULT %s", val));
 
-      g_ptr_array_add (new_lines, g_strdup_printf ("LABEL %s", val));
+      g_ptr_array_add (new_lines, g_strdup_printf ("\nLABEL %s", val));
```

Comment:
The way we're adding newlines into what is intended to be an "array of lines" feels...slightly wrong though it's clearly going to work.  But WDYT about:
```
      g_ptr_array_add (new_lines, g_strdup (""));
      g_ptr_array_add (new_lines, g_strdup_printf ("LABEL %s", val));
```
instead?